include/vision.h

```diff
@@ -32,7 +32,7 @@ class Vision
 
   void run();
   void quit();
-  void config_rgb_camera();
+  void config_rgb_camera(std::string camera_name, std::string frame_id);
```

Comment:
I was going to say that it is preferable to pass by const reference (i.e. `const std::string &camera_name`), but it seems that might not always be the case in C++11. See [here](https://stackoverflow.com/questions/2139224/how-should-i-pass-objects-to-functions) and [here](https://codesynthesis.com/~boris/blog/2012/06/19/efficient-argument-passing-cxx11-part1/).